### ISSUE TITLE
Issue : I can't enter the shop

### ISSUE THREAD

**Issue by maximaxxxx** [AbsoluteStratos/ChristmasInDirtmouth#3]
![image](https://github.com/user-attachments/assets/810b00ba-00d2-431c-865d-65a934bb2a65)

I can push all the button of my keyboard, the knight refuse to enter the shop. I've try to restart the game, uninstall, install the mod, change my save... So.... please help 😅

**Comment by AbsoluteStratos** [comment]
Hi @maximaxxxx 

Thanks for the report. The enter key should be the "up" input like the other shops.
Are you using any other mods with it? What OS are you on?

I'm assuming since the screen shop has the arrow prompt that it appears, animates and disappears like the other shops correct?

**Comment by maximaxxxx** [comment]
yes, the prompt appears and disapears. I've try all the key of the keyboard ^^ and i'm using windows 11 and few other mods like hkmp, benchwarp or debug mods (not vocalized)

**Comment by AbsoluteStratos** [comment]
Thanks @maximaxxxx , appreciate the information.

I cant reproduce myself but added an additional input checkout for the "Up" press. Update to `1.0.2` and try pressing Up while the door is active.
Additionally added a fallback (Key press O) for the door and end event incase is still doesnt work. Not sure why this is the case for you at the moment.

Will be on mod links once merged: https://github.com/hk-modding/modlinks/pull/1628
Otherwise install manually via the zip: https://github.com/AbsoluteStratos/ChristmasInDirtmouth/releases/tag/v1.0.2

Thanks!

**Comment by maximaxxxx** [comment]
your welcome ^^
